app/components/shared/card_component.rb

```diff
@@ -0,0 +1,41 @@
+class Shared::CardComponent < GovukComponent::Base
+  include ViewComponent::SlotableV2
+
+  renders_many :action_items, "ActionItemComponent"
+  renders_many :body_items, "ItemComponent"
+  renders_many :header_items, "ItemComponent"
```

Comment:
Can you persuade me that we need two methods to access the same component?

Reply:
because the component does exactly the same thing but the items for each are rendered in different parts of the component